README.md

```diff
@@ -1,41 +1,30 @@
-# [WIP] skupper [WIP]
+# Skupper
+Skupper enables cloud communication by enabling you to create a Virtual Application Network.
 
-Note: this repository is a work in progress targeted for the consolidation
-and refactoring of the skupper implementation.
+This upper layer network decouples addressing from the underlying network infrastructure, providing a secure and simple alternative to VPNs.
 
-Command line tool for setting up and managing skupper installations
+You can use Skupper to create simple integrations between clusters as described in the [Getting Started](https://skupper.io/start/index.html) or create a topology that supports:
 
-## Usage
+* Redundant paths
+* Cross-network multicast
+* Cross-network anycast for load balancing
 
-See `skupper help` or `skupper <command> --help` for details.
+Skupper does not require any special privileges, that is, you do not require the `cluster-admin` role to create networks.
 
-## Example
+# Useful Links
+Using Skupper
 
-In one kubernetes context do:
+* [Getting Started](https://skupper.io/start/index.html)
+* [Examples](https://skupper.io/examples/index.html)
+* [Documentation](https://skupper.io/docs/index.html)
 
-```
-skupper init
-skupper connection-token /path/to/mysecret.yaml
-```
-
-In another context, e.g. another kubernetes cluster, do:
-
-```
-skupper init
-skupper connect --secret /path/to/mysecret.yaml
-```
-
-By default skupper will try to set itself up to allow connections from
-other skupper sites (using mutual TLS). It uses a LoadBalancer service
-or an OpenShift Route for this. If you don't want this, or your
-cluster is not set up to support those options, you can use the
-`--cluster-local` option to `skupper init` and will then be able only
-accept connections from skupper instances in different namespaces on
-the same cluster.
-
-Note: if using minikube, you can get the LoadBalancer service to setup
-an external ip by running minikube tunnel. (Or else use
---cluster-local as described above).
 
+Developing Skupper
 
+* [Community](https://skupper.io/community/index.html)
+* [Site controller](cmd/site-controller/README.md)
+* [CLI](cmd/skupper/README.md)
+* [Console](/skupperproject/gilligan)
 
+# Licensing
+Skupper is related to the [Apache QPID Dispatch Router](https://github.com/apache/qpid-dispatch) project and uses the same [Apache License 2.0](https://github.com/skupperproject/skupper/blob/master/LICENSE).
```

Comment:
Rather than the vague 'is related to', perhaps 'uses' (and then say 'and is released under the same AL2 license' to avoid repetition of 'uses')?